stdlib/REPL/test/replcompletions.jl

```diff
@@ -2650,3 +2650,17 @@ let s = "\"example: \$(named.len"
     @test "len2" in c
     @test r == 19:21
 end
+
+# #58296 - complete positional arguments before semicolon
+let s = "string(findfi|; base=16)"
+    c, r = test_complete_pos(s)
+    @test "findfirst" in c
+    @test r == 8:13
+end
+
+# Unknown functions should not cause completions to fail
+let s = "foo(findfi"
```

Comment:
`foo` is a popular name here, we had problems in the past because of this habit: #53110. Mangling names with issue numbers should reduce the probability of clashes and ensure you're testing what you meant to (using an undefined function):
```suggestion
let s = "foo58296(findfi"
```